### PR TITLE
integrate keyword mapping to explorer server

### PR DESCRIPTION
**Pull request by Tilana**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM python:3.6.5
 
 COPY ./app /flask-explorer/app
 COPY ./server.py /flask-explorer/server.py
+COPY ./models /flask-explorer/models
```
Comment (todorus) [inline Dockerfile]: the model shouldn't be baked into the image, but we should rather use a volume for that. That way we can just switch out model files

File: app/__init__.py [hunk 2]
```diff
@@ -12,8 +13,7 @@ def create_app(model_path=None):
     if model_path is None:
```
Comment (todorus) [inline app/__init__.py]: Could you make an environment variable out of this? Then we'll be able to configure containers

**Comment by todorus** [comment]
There are some binary files changed. I put some in there as a proof of concept, but we should try to keep only code in git repos. Git isn't made for big files and it will become slow pretty quickly. 

I can't tell if these files are just there as a proof of concept, to show how the application works. That would be fine, but we should keep our model files seperate.

File: Dockerfile (hunk 1 above)
Comment (Tilana) [reply Dockerfile]: yes, indeed!

File: app/__init__.py (hunk 2 above)
Comment (Tilana) [reply app/__init__.py]: I added a config file with the path to the default model set

**Comment by Tilana** [comment]
For testing purposes I added a functioning model to the repos such that just downloading the repo provides users with all necessary components...
The binary file is different as I changed the way of storing and loading the models to save disk space...

**Pull request merged**